packages/landing/src/components/feature.updates.tsx

```diff
@@ -0,0 +1,140 @@
+import { clsx } from 'clsx';
+import { Component, ReactNode } from 'react';
+import ReactModal from 'react-modal';
+
+interface IFrameConfig {
+  url: string;
+  width: number;
+  height: number;
+  title: string;
+  iframeWrapperClass?: string;
+  iFrameProps?: Partial<JSX.IntrinsicElements['iframe']>;
+}
+
+type FeatureUpdatesProps = {
+  header: string;
+  wrapperClass?: string;
+  id: string;
+  releaseVersion: string;
+  enabled: boolean;
+  children?: ReactNode;
+} & (
+  | { bigImage: string; smallImage: string; iframe?: never }
+  | { bigImage?: never; smallImage?: never; iframe: IFrameConfig }
+);
+
+type FeatureUpdatesState = {
+  showModal: boolean;
+};
+
+/**
+ * FeatureUpdates is a re-implementation of @linzjs/lui -> LuiUpdatesSplashModal module,
+ * This use to enable a one off pop up screen for introduce new feature of the recent release.
+ *
+ */
+export class FeatureUpdates extends Component<FeatureUpdatesProps, FeatureUpdatesState> {
+  constructor(props: FeatureUpdatesProps) {
+    super(props);
+    const currentVersion = window.localStorage.getItem(this.props.id);
+
+    this.state = {
+      showModal: this.props.enabled && this.props.releaseVersion.trim() !== currentVersion,
```

Comment:
do we have a requirement that releaseversion limits when this is shown?

Reply:
Not in the Acceptance Criteria. But I think [STEP is using split io](https://github.com/linz/landonline-search/blob/c1f02a4e2ec427c211896495bea4a886bc92f7ea/web/src/components/RecentUpdates.tsx#L15) for the control. We only limited by an end date.

Reply:
version is not a date?

Reply:
The `releaseVersion` basically just a cache of `dissmissKey` in lui.
I have put the date check in the `[new.feature.tsx](https://github.com/linz/basemaps/pull/3356/files#diff-ed4cda0d05e45a43654131c1ff2f93f88f4c0a7e7f7e84471b1b52d9c8b94077)` line 32.

Reply:
I have removed the releaseVersion and only add `dissmissKey` and `closeDate` for now.